### PR TITLE
Feat: Toolbox experience improvements

### PR DESCRIPTION
**Pull request by tbwiss**
# Description

New:
- Keybindings are now shown in the Help&Learn drawer
- Possibility to select a database for multi-db environments
- Ability to toggle lint markers (the red dots) for the graphql editors, it's also a stored setting
- Use the correct Canny link

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [ ] Documentation has been updated
- [ ] TCK tests have been updated
- [ ] Integration tests have been updated
- [ ] Example applications have been updated
- [ ] New files have copyright header
- [ ] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the UI!

Check out your changes at
 http://neo4j-graphql-toolbox-stage-1413.surge.sh (=> for local development; use for instance [Neo4j Desktop](https://neo4j.com/download/))
or
 https://neo4j-graphql-toolbox-stage-1413.surge.sh (=> use to connect to an [Aura Free](https://neo4j.com/cloud/aura-free/) dbms)

**Comment by darrellwarde** [comment]
I'm not a React guru so won't do a lot of code review here, but have some visual feedback!

Screenshot 2022-05-15 at 20 32 25
Having radio buttons next to checkboxes feels a bit unusual, and also quite unusual that one of the radio buttons isn't selected by default. Perhaps a small header "Constraints" with radio buttons "Ignore", "Check", "Create", or a dropdown box perhaps?

Screenshot 2022-05-15 at 20 32 36
Shall we rename this file `type-definitions.graphql`?

Screenshot 2022-05-15 at 20 32 45
I'm not sure if I like "typeDefs" in this button - perhaps "Generate type definitions", or just "Generate"?

Screenshot 2022-05-15 at 20 33 08
To me, it's not clear that the database selection is a dropdown box, perhaps some faint outlining here?

Screenshot 2022-05-15 at 20 33 19
Similar to above, unclear to me that the icon and text are the same button, worth some faint outlining around them?

Other than the above, this is looking super slick, I can't wait for our users to try it out!

**Comment by tbwiss** [comment]
Thanks for the review @darrellwarde! Implemented the changes.
I like all the suggestions but I'm not sure about the "Disconnect" button. Not sure I'm in favour of it. it's added for now at least, have a look and let me now.

**Comment by tbwiss** [comment]
Asked for Liza's input regarding the Disconnect button. Included that in https://github.com/neo4j/graphql/pull/1413/commits/c0667626941a8fe4c5d0c3955322c0402c92d4ba

File: packages/graphql-toolbox/src/contexts/appsettings.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { LOCAL_STATE_CONSTRAINT, LOCAL_STATE_SHOW_LINT_MARKERS } from "src/constants";
+import { ConstraintState } from "src/types";
+import { Storage } from "../utils/storage";
+
+export interface State {
+    showLintMarkers: boolean;
+    setShowLintMarkers: (v: boolean) => void;
+}
+
+export const AppSettingsContext = React.createContext(null as unknown as State);
```
Comment (angrykoala) [inline packages/graphql-toolbox/src/contexts/appsettings.tsx:30]: Is triple casting the best way of achieving this? :sweat_smile:

**Review by angrykoala — COMMENTED** [review]
LGTM, not much feedback I can give regarding the react code though

File: packages/graphql-toolbox/src/contexts/appsettings.tsx (hunk 1 above)
Comment (tbwiss) [reply packages/graphql-toolbox/src/contexts/appsettings.tsx:30]: Good old copy&pasting.. Fixing this for all contexts

**Comment by tbwiss** [comment]
If there are no more comments or updates on this then I'll merge it later today

**Pull request merged**